### PR TITLE
Web ide code review

### PR DESCRIPTION
**Pull request by dennisbuitendijk**

### CODE REVIEW

File: dockerspawner/dockerspawner.py [hunk 1]
```diff
@@ -965,7 +883,9 @@ async def get_command(self):
         else:
             image_info = await self.docker("inspect_image", self.image)
             cmd = image_info["Config"]["Cmd"]
-        return cmd + self.get_args()
+            args = self.get_args()
+        return cmd #+ ['--auth=none']
+        #+ self.get_args()
```
Comment (ArquintL) [inline dockerspawner/dockerspawner.py]: Is this change intentional? You get the arguments but you do not do anything with them. Also, please remove commented out code

**Review by ArquintL — CHANGES_REQUESTED** [review]
One minor question, otherwise looks good

File: dockerspawner/dockerspawner.py (hunk 1 above)
Comment (ArquintL) [reply dockerspawner/dockerspawner.py]: Why are the `args` no longer necessary? Does the super implementation provide some that are unsuitable?

File: dockerspawner/dockerspawner.py [hunk 2]
```diff
@@ -1182,6 +1100,40 @@ async def start(self, image=None, extra_create_kwargs=None, extra_host_config=No
...
 
+        #add code-server config with custom password located in self.user.server_passwd
+        docker_client = docker.from_env()
+        container = docker_client.containers.get(self.container_id)
+        with open('/srv/jupyterhub/configs/config_{}.yaml'.format(self.user.escaped_name), 'w') as f:
+            f.write("""bind-addr: 127.0.0.1:8080
+auth: password
+password: {}
+cert: false
+""".format(self.user.server_passwd))
```
Comment (ArquintL) [inline dockerspawner/dockerspawner.py]: Indentation looks ugly (I know it's not your fault). However the new [`f` strings](https://realpython.com/python-f-strings/#multiline-f-strings) might be a better alternative (I think you need to explicitly insert `\n` to create a line break).

**Review by ArquintL — CHANGES_REQUESTED** [review]

File: dockerspawner/dockerspawner.py (hunk 1 above)
Comment (dennisbuitendijk) [reply dockerspawner/dockerspawner.py]: The implementation adds flags that are not supported by code-server. They added those for the communication between the Hub and the Notebooks. We already have all the flags and arguments needed for code-server in our dockerfile, from where this implementation also gets the command.

**Review by ArquintL — APPROVED** [review]
